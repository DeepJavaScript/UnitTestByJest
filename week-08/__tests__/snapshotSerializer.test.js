import serializer from '../src/my-serializer-module';

expect.addSnapshotSerializer(serializer);

it('snapshot object', () => {
  const bar = {
    foo: {
      x: 1,
      y: 2,
    },
  };

  expect(bar).toMatchSnapshot();
});