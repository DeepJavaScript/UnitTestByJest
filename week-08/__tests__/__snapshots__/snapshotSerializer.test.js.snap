// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot object 1`] = `
Object {
  "foo": Object {
    "x": 1,
    "y": 2,
  },
}
`;
